test/unit/pusher_test.rb

```diff
@@ -263,16 +263,16 @@ class PusherTest < ActiveSupport::TestCase
       @cutter.stubs(:rubygem).returns @rubygem
       create(:version, rubygem: @rubygem, number: '0.1.1', summary: 'old summary')
       @cutter.stubs(:version).returns @rubygem.versions[0]
-      @rubygem.stubs(:update_attributes_from_gem_specification!)
-      GemCachePurger.stubs(:call)
+      @rubygem.expects(:update_attributes_from_gem_specification!).with(@cutter.version, nil)
+      GemCachePurger.expects(:call).with(@rubygem.name).at_least_once
       Indexer.any_instance.stubs(:write_gem)
+
       @cutter.save
     end
 
     should "update rubygem attributes" do
-      assert_received(@rubygem, :update_attributes_from_gem_specification!) do |rubygem|
-        rubygem.with(@cutter.version, @cutter.spec)
-      end
+      @rubygem.expects(:update_attributes_from_gem_specification!).with(@cutter.version, nil)
```

Comment:
Please don't use `nil`. I think `setup` is missing spec:
```
@spec = mock
@cutter.stubs(:spec).returns @spec
```
and this line should be `@rubygem.expects(:update_attributes_from_gem_specification!).with(@rubygem.versions[0], @spec)`